### DISCUSSION TITLE
Linguist is not detecting HTML in my repository

### DISCUSSION THREAD

**Discussion by chasegalloway** [github-linguist/linguist#6791] [Classification] [answered]
Linguist is failing to recognize HTML files in a repository of mine(well technically my organization's, but I still own it), despite the fact that there are very clearly .html files within the directory.

**Comment by lildude** [comment, marked as answer]
Got a link we can look at?  We can't provide a precise explanation without seeing it in action.

You might want to also take a look at https://github.com/github-linguist/linguist/blob/master/docs/how-linguist-works.md for all the things Linguist ignores... it's possible your HMTL files are in one of the paths Linguist considers documentation.

**Reply by chasegalloway** [reply]
That was the problem. The name of the folder that I kept the html files in was named 'docs', which made Linguist consider the html files to be documentation. I changed the folder's name and that fixed the problem, thank you!

**Reply by lildude** [reply]
You can also keep the folder named `docs` and implement an override, this one specifically: https://github.com/github-linguist/linguist/blob/master/docs/overrides.md#documentation

**Reply by chasegalloway** [reply]
Thank you, I have done that and everything is working properly.
